python/hsfs/expectation_suite.py

```diff
@@ -48,18 +54,39 @@ def __init__(
     ):
         self._id = id
         self._expectation_suite_name = expectation_suite_name
-        self._featurestore_id = featurestore_id
-        self._featuregroup_id = featuregroup_id
         self._ge_cloud_id = ge_cloud_id
         self._data_asset_type = data_asset_type
         self._run_validation = run_validation
         self._validation_ingestion_policy = validation_ingestion_policy.upper()
         self._expectations = []
+        self._href = href
+
+        if href is not None:
+            self._init_feature_store_and_feature_group_ids_from_href(href)
```

Comment:
I don't understand why this is necessary, when you initialize all the objects, iterating through the "items" of the RestDTO, they should all have the feature store and group id set?

Reply:
let's follow up in another Jira